schedule/yast/yast2_gui.yaml

```diff
@@ -25,3 +25,6 @@ schedule:
     - yast2_gui/yast2_software_management
     - yast2_gui/yast2_users
     - yast2_gui/yast2_security
+    - yast2_gui/yast2_lan_restart_bridge
```

Comment:
As we had this test running everywhere, so by moving it to the `yast2_gui` decreases the coverage, so we need to enable it for more scenarios. It doesn't mean that we need to run it everywhere, but 64-bit only is not enough I would say.